backend/compact-connect/lambdas/python/provider-data-v1/tests/function/test_handlers/test_provider_users.py

```
@@ -376,3 +383,947 @@ def test_patch_provider_military_affiliation_updates_status(self):
+
+    def test_put_provider_home_jurisdiction_updates_privileges_for_multiple_license_types_in_new_jurisdiction(self):
+        from cc_common.data_model.schema.privilege import PrivilegeData
+        from handlers.provider_users import provider_users_api_handler
+
+        """
+        In this test case, the user has two licenses in the current jurisdiction, and two matching licenses in the new
+        jurisdiction when the user updates the home jurisdiction selection. The privileges should be moved over to their
+        respective license types.
+        """
+
+        (test_provider_record, test_current_license_record_type_one, test_privilege_record_license_type_one) = (
```

Pylint (3.3.7)

[convention] Line too long (112/100)

(C0301)